LayoutTests/svg/custom/elementTimeControl-nan-crash.html

```diff
@@ -2,14 +2,15 @@
 <html>
 <head>
 <script>
+    if (window.testRunner)
+        testRunner.dumpAsText();
+
     function crash() {
         var animate = document.getElementById('animate');
         var svg = document.getElementById('svg');
-        animate.endElementAt(NaN);
-        animate.beginElementAt(NaN);
+        try { animate.endElementAt(NaN); } catch (e) {}
+        try { animate.beginElementAt(NaN); } catch (e) {}
```

Comment:
I think it is better not to silence the exceptions. Please regenerate new expectations with the exceptions messages and keep this test as is.

Reply:
I did this change post your feedback in previous PR about this:
```Why do you want to make the expected file a dump render tree instead of dump as text? Also I do not see the changes in the test file LayoutTests/svg/custom/elementTimeControl-nan-crash.html to make it output this result.```

Reply:
@shallawa - Happy to revert and do platform specific expectation. I did this in my previous PR: [Link](https://github.com/WebKit/WebKit/pull/8660/files#diff-a9c09919ea260012874792863da1eba363af7acc7b5487c3d4e47e8aa790475f)

Reply:
Your approach looks fine as well since this test was for a crash. So it is okay to ignore the exceptions.

Reply:
Thanks!